tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -2636,6 +2637,8 @@ def spatial_softmax(features,
   Args:
     features: A `Tensor` of size [batch_size, W, H, num_channels]; the
       convolutional feature map.
+    activation_fn: Activation function. The default value is a softmax function.
+      Explicitly set it to None to skip it and maintain a linear activation.
```

Comment:
I'm not sure if linear activations would work. Maybe just comment that softmax and sigmoid can work well.

Reply:
If I recall correctly, I made the change this way because "linear" would be used in the case where you're getting the logits out, and you want to apply your own custom algorithm on the output without applying an activation yet.